package.json

```diff
@@ -9,20 +9,19 @@
   },
   "scripts": {
     "build": "ember build",
-    "coverage": "COVERAGE=true ember test -r 'dot'",
+    "test": "npm run lint && COVERAGE=true ember test",
```

Comment:
I didn't actually manually change any ordering in any file. Big alphabetical fan here. Not sure which merge this came from